### DISCUSSION TITLE
Are there any other features we should add?

### DISCUSSION THREAD

**Discussion by bonomo1** [CorterraSolutions/SolutionDependencyChecker#10] [Ideas]
After realizing some people were using this other than ourselves, are there any other features we should consider adding?

**Comment by mxwsxr** [comment]
Hi please add a feature where I can see in XrmToolBox 2 different views. On one side I wanna see 1 CRM environment and the other side for example the Test environment. Both views have a list of solutions listed as they are. When I select one  solution on the left and one on the right it should show me in the result section if they have dependencies or not.

**Comment by bonomo1** [comment]
Thanks for the idea. We will keep it in mind any future changes to compare solutions between environments within the user interface

**Reply by mxwsxr** [reply]
Or maybe simples: If it's possible to implement it asap :)

For example I am on DEV environment. I have many patch solutions. Before deployment I wanna compare 2 patches for dependencies OR I choose 1 patch and run dependency checking and it should return me a list of patches which have dependencies to this chosen patch.
Would that be possible?

**Reply by bonomo1** [reply]
Good idea. I think that would be possible but I am unsure how easily it could be added to this solution in its current state. I'd suggest a separate tool.

Are you a member of the XRM Toolbox Slack group? There is a channel for tool suggestions on there.
